stixcore/util/scripts/rebuild_hook.cgi

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/bash
```

Comment:
Yea I think what you probably needed was the cgi module e.g. (https://stackoverflow.com/questions/44871139/how-do-i-run-python-cgi-script-on-apache2-server-on-ubuntu-16-04) to connect the python to the web server but this works too.